peribolos/knative-sandbox.yaml

```diff
@@ -637,6 +637,17 @@ orgs:
             - devguyio
             - lionelvillard
             privacy: closed
+          Source WG Leads:
```

Comment:
This team does not exist anymore as the Source WG has been merged back into the Eventing WG